e2e/typescript_proto_usage/tests/extending_jasmine/extending.spec.js

```diff
@@ -0,0 +1,7 @@
+import { ExampleMessage } from "../../extending_pb.mjs"
+
+describe("Extending proto", () => {
+  it("should have example message defined", () => {
+    expect(new ExampleMessage()).not.toBeUndefined();
+  });
+});
```

Comment:
Optional: Consider adding a test case that imports [`operations.proto`](https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto) since that's what broke for us. It may exercise more code paths.

Reply:
modified the `extending.proto` file to not only directly use the `extend` keyword but also to import `operations.proto` as a field in the `ExampleMessage`